src/Compilers/VisualBasic/VbcCore/VbcCore.csproj

```diff
@@ -15,7 +15,7 @@
     <LargeAddressAware>true</LargeAddressAware>
     <StartupObject>Microsoft.CodeAnalysis.VisualBasic.CommandLine.Program</StartupObject>
     <SolutionDir Condition="'$(SolutionDir)' == '' OR '$(SolutionDir)' == '*Undefined*'">..\..\..\..\</SolutionDir>
-    <OutDir>$(OutDir)core-clr\</OutDir>
+    <OutDir>$(OutDir)$(CoreClrTargetFolder)</OutDir>
```

Comment:
I don't understand why we would want to do this.  The actual binary we produce is platform independent.  Why would we emit it to a platform specific directory?